src/sdk/namespace_fs.js

```diff
@@ -681,6 +694,15 @@ class NamespaceFS {
                 }
             };
             for await (let data of source_stream) {
+                // Update the write stats
+                let count = 1;
```

Comment:
Why do you need to varible?
You do not increase it or manipulate it except assigning 0 that will get overridden back next cycle to 1. I do not understand this flow.

Reply:
This flow is Exact as the other namespaces. 
I rather keep it the same.
If we want we can refractor all later.

Reply:
It is not the same behavior because you send 1 for every data event.
I assume that you want to see how many files you've uploaded/read, not the number of buffers you passed.